testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -1,5 +1,34 @@
 # Copyright (c) 2010-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
+#
+# This feature can cause failures in:
+# - features/init_client/buildhost_bootstrap.feature
+# - features/init_client/sle_minion.feature
+# - features/secondary/buildhost_docker_auth_registry.feature
+# - features/secondary/buildhost_docker_build_image.feature
+# - features/secondary/buildhost_osimage_build_image.feature
+# - features/secondary/min_baremetal_discovery.feature
+# - features/secondary/min_bootstrap_api.feature
+# - features/secondary/min_bootstrap_reactivation.feature
+# - features/secondary/min_bootstrap_script.feature
+# - features/secondary/min_docker_api.feature
+# - features/secondary/min_move_from_and_to_proxy.feature
+# - features/secondary/min_salt_mgrcompat_state.feature
+# - features/secondary/min_salt_minions_page.feature
+# - features/secondary/minkvm_guests.feature
+# - features/secondary/proxy_as_pod_basic_tests.feature
+# - features/secondary/proxy_cobbler_pxeboot.feature
+# - features/secondary/proxy_retail_pxeboot.feature
+# - features/secondary/srv_docker_advanced_content_management.feature
+# If the SUSE-KEY-x86_64 fails to be created
```

Comment:
can you reorder these please so the If statement is in the beginning and what will fail after it? It's weird to read this now